src/AddTodoForm.js

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import InputWithLabel from './InputWithLabel.js';
+
+
+
+const AddTodoForm = ({onAddTodo}) => {
+  const [todoTitle, setTodoTitle] = React.useState('');
+  const handleTitleChange = (event) => {
+    let newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  }
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+
+
+  //  console.log(todoTitle);
+
+    onAddTodo({
+      id: Date.now(),
+      title: todoTitle,
+    })
+    setTodoTitle('')
+  };
+  return(
+
+
+
+    <form onSubmit={handleAddTodo}>
+    <InputWithLabel
+
+      id="search"
+      //value={todoTitle}
+      todoTitle={todoTitle}
+      handleTitleChange={handleTitleChange}
+      //onInputChange={handleTitleChange}
+    >
+    <strong>Search:</strong>
```

Comment:
Is "Search" an accurate label?

Reply:
I think I made all the required changes.